### PR TITLE
Creating files to start the API

### PR DESCRIPTION
**Pull request by KevinCastroP**
I just added this commit please review

### CODE REVIEW

File: api/v1/app.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/python3
+"""
+Module status
+"""
+
+
+from flask import Flask, jsonify, Blueprint, make_response
```
Comment (diesgaro) [inline api/v1/app.py:7]: Why import jsonify, Blueprint and make_response???

File: api/v1/app.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+@app.teardown_appcontext
+def teardown_db(response_or_exc):
+    """Method to close the session"""
+    storage.close()
+
+if __name__ == '__main__':
+    app.run(host=getenv('HBNB_API_HOST'),
+            port=getenv('HBNB_API_PORT'),
+            threaded=True)
```
Comment (diesgaro) [inline api/v1/app.py:24]: A white space is missing

File: api/v1/app.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/python3
+"""
+Module status
```
Comment (diesgaro) [inline api/v1/app.py:3]: Module status?? This script is general for the next methods, please make a better documentation

File: api/v1/views/__init__.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/python3
+"""
+Initial Blueprint
+"""
+
+
+from flask import Flask, Blueprint
+from api.v1.views.index import *
+
+app_views = Blueprint("app_views", __name__, url_prefix="/api/v1")
```
Comment (diesgaro) [inline api/v1/views/__init__.py]: The white space is missing

File: api/v1/views/__init__.py [hunk 5]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/python3
+"""
+Initial Blueprint
+"""
+
+
+from flask import Flask, Blueprint
```
Comment (diesgaro) [inline api/v1/views/__init__.py]: why import Flask???

File: api/v1/views/index.py [hunk 6]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/python3
+"""
+Creating an Index
+"""
+
+
+from api.v1.views import app_views
+from flask import Flask, jsonify
```
Comment (diesgaro) [inline api/v1/views/index.py:8]: Why import Flask??

File: api/v1/views/index.py [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+from models import storage
```
Comment (diesgaro) [inline api/v1/views/index.py:9]: Why import storage???

File: api/v1/views/index.py [hunk 8]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/python3
+"""
+Creating an Index
```
Comment (diesgaro) [inline api/v1/views/index.py]: Make better documentation please

File: api/v1/views/index.py [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+
+
+from api.v1.views import app_views
+from flask import Flask, jsonify
+from models import storage
+
+@app_views.route('/status', strict_slashes=False)
+def status():
+    """Checking the status"""
+    return jsonify(status='OK')
```
Comment (diesgaro) [inline api/v1/views/index.py:14]: white space is missing

**Review by diesgaro — COMMENTED** [review]
Please work in the corrections, meke better documentation and make sure using pycodestyle/PEP8 in your code.

File: api/v1/views/__init__.py (hunk 4 above)
Comment (diesgaro) [reply api/v1/views/__init__.py]: This code has a error, please view the requirements for this file

Comment (KevinCastroP) [reply api/v1/views/__init__.py]: ok, I'm going to make the changes

File: api/v1/views/__init__.py (hunk 5 above)
Comment (KevinCastroP) [reply api/v1/views/__init__.py]: that was an error, I'm going to change that and fix it

File: api/v1/views/__init__.py [hunk 10]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/python3
+"""
+Initial Blueprint
+"""
+
+
+from flask import Blueprint
+app_views = Blueprint("app_views", __name__, url_prefix="/api/v1")
+from api.v1.views.index import *
```
Comment (diesgaro) [inline api/v1/views/__init__.py:9]: white space is missing

File: 7-dump.sql [hunk 11]
```diff
@@ -0,0 +1,95 @@
+-- MySQL dump 10.13  Distrib 5.7.8-rc, for Linux (x86_64)
```
Comment (diesgaro) [inline 7-dump.sql]: This file is not necesary

Comment (KevinCastroP) [reply 7-dump.sql]: R, I delete that file

File: api/v1/views/__init__.py (hunk 10 above)
Comment (KevinCastroP) [reply api/v1/views/__init__.py:9]: thats the syntax for this file

File: api/v1/views/index.py (hunk 9 above)
Comment (KevinCastroP) [reply api/v1/views/index.py:14]: thats the syntax for this type files

File: api/v1/views/index.py (hunk 8 above)
Comment (KevinCastroP) [reply api/v1/views/index.py]: ok, I'm going to change that comment

File: api/v1/views/index.py (hunk 6 above)
Comment (KevinCastroP) [reply api/v1/views/index.py:8]: yes, that was an error is not necesary

File: api/v1/views/index.py (hunk 7 above)
Comment (KevinCastroP) [reply api/v1/views/index.py:9]: Because is necesary to the execution of the file app.py to import all the modules and methods to run the file

**Comment by KevinCastroP** [comment]
I just finish to fixing the pep8 errors

**Pull request merged**